docs/clients/p4rt-ctl.rst

```diff
@@ -319,7 +319,61 @@ Details about the supported commands
     ``COUNTER_FLOW``: Refers to the counter ID (generated ID by p4c, refer to bfrt.json file)
     and the counter table index. Format "counter_id=<number>,index=<number>"
 
-16. Get flow dump entries ::
+
+16. Add a rule for direct counter into the forwarding pipeline::
```

Comment:
Please keep the format consistent, similar to line 259 - "Rule to program direct counter?

Reply:
yes